test/controllers/declaration/DeclarationHolderControllerSpec.scala

```diff
@@ -284,20 +284,6 @@ class DeclarationHolderControllerSpec
           verifyTheCacheIsUnchanged()
         }
       }
-
-      "try to remove not added Additional code" in {
```

Comment:
What about this case?

Reply:
I dont see why it matters if we try and remove something that isnt there already. The outcome is still the same in that "the thing you asked us to remove doesnt exist" after the remove operation is complete.

If we can reduce the things that can lead a user to the "error!" page I think we should